READMEV2.md

```diff
@@ -0,0 +1,76 @@
+# VEDA Features API
+
+Hosting and serving collections of vector data features for VEDA
+
+---
+
+## Implementation
+
+* Storage: PostGIS
+* WFS3 API with query support: [OGC API Features](https://ogcapi.ogc.org/features/) provided by [TiPG](https://github.com/developmentseed/tipg)
+* Vector tiles API provided by [TiPG](https://github.com/developmentseed/tipg)
+
+---
+
+### Local Development in Docker
+
+To locally run the site:
+
+`docker-compose up`
+
+---
+
+### Continuous Deployment for `staging` and `production`
```

Comment:
reviewer note: I think `us-west-2` is completely out of elastic IPs. So I need to talk with other folks in data services to assess if we need to ask for an increase or are fine using `us-west-1`. So anywhere in the docs where we call out AWS "regions" as a hard dependency on where we are installing things might just